javatests/arcs/showcase/typevariables/Consumer.kt

```diff
@@ -0,0 +1,25 @@
+package arcs.showcase.typevariables
+
+import com.google.common.truth.Truth.assertThat
+import kotlinx.coroutines.Job
+
+class Consumer : AbstractConsumer() {
+
+    /** Process an order. In this case, we ensure skus have been redacted and prices are fair. */
+    override fun onUpdate() =
+        handles.data.fetchAll()
+            .also { assertThat(it).hasSize(3) }
```

Comment:
As discussed elsewhere in this PR, assertThat() should not be used in particles (ideally)